gapic-generator-cloud/bin/ruby-cloud-docker-entrypoint

```diff
@@ -72,8 +88,11 @@ OptionParser.new do |op|
   map_option_map.each do |key, flags|
     flags = Array(flags).map { |f| "--#{f} MAPPING" }
     op.on(*flags) do |mapping|
-      mapping_key, mapping_val = mapping.split("=", 2)
-      parameters["#{key}.#{mapping_key}"] = mapping_val if mapping_val
+      mapping.split(";").each do |mapping_kv|
+        mapping_key, mapping_val = mapping_kv.split("=", 2)
+        mapping_key.gsub!(".", "\\\\\\\\.")
```

Comment:
I know. It's terrible. 2**3 backslashes to insert one. Because:
* One instance of escaping because the syntax for the protoc parameter requires backslashes to be escaped
* One instance of escaping because it's a double-quoted string.
* One instance of escaping because gsub interprets backslashes specially (to denote back-references) so literal backslashes need to be escaped.

The world hurts...